packages/web/components/Technology/Details/Tabs/index.js

```diff
@@ -21,31 +21,36 @@ const Tabs = () => {
 	];
 
 	return (
-		<Container>
-			<TabList>
+		<StyledTabs>
+			<StyledTabList>
 				{tabs.map((tab) => (
-					<Tab key={tab.slug} data-testid={tab.slug}>
+					<StyledTab key={tab.slug} data-testid={tab.slug}>
 						{tab.label}
-					</Tab>
+					</StyledTab>
 				))}
-			</TabList>
+			</StyledTabList>
 
 			{tabs.map((tab) => (
 				<TabPanel key={tab.slug}>
 					<Row>{React.createElement(tab.component)}</Row>
 				</TabPanel>
 			))}
-		</Container>
+		</StyledTabs>
 	);
 };
 
+/**
+ * This isn't Next.JS getInitialProps
+ */
 Tabs.getInitialProps = () => {
 	resetIdCounter();
 };
```

Comment:
Maybe that's why it doesn't work. Can you put this code in the page component?

Reply:
@mateus4k you call this method in some tests.  I couldnt understand your trakinagem

Reply:
Thanks for your contribution